### ISSUE TITLE
GCC 10 requires CFLAGS=-fcommon for build

### ISSUE THREAD

**Issue by wyatt8740** [dimkr/gtk#1]
GCC 10 is a little stricter about redefining things in headers; GTK 1.2 fails to build unless '-fcommon' is passed. The same is true for glib.

**Comment by dimkr** [comment]
Thanks for reporting @wyatt8740. These projects are unmaintained, and I can't promise this will be fixed anytime soon.

**Comment by wyatt8740** [comment]
yeah, I assumed as much, I'm thankful regardless. I just wanted this somewhere where others could see it in case they're getting frustrated.
